library/std/src/process.rs

```diff
@@ -1038,6 +1038,13 @@ impl fmt::Debug for Command {
     /// Format the program and arguments of a Command for display. Any
     /// non-utf8 data is lossily converted using the utf8 replacement
     /// character.
+    ///
+    /// The default format approximates a shell invocation of the program along with its
+    /// arguments. It does not include most of the other command properties. The output is not guaranteed to work
+    /// (e.g. due to lack of shell-escaping or differences in path resolution)
+    /// On some platforms you can use [the alternate syntax](https://doc.rust-lang.org/std/fmt/index.html#sign0) to show more fields.
```

Comment:
This should be an [intra-doc link](https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html#valid-links) so that locally generated docs work too.
And the reference form (with the definition below the paragraph) is preferred in std.

Reply:
done 👍